### PR TITLE
Check product minimal quantity

### PR DESCRIPTION
**Pull request by Rmagic96**
The module doesn't check minimal quantity of product or product attributes when sends an email to suscribed customers. 
This fix modifies this and now checks the minimal quantity instead of just checking if the quantity is greater than 0.

### CODE REVIEW

File: ps_emailalerts.php [hunk 1]
```diff
@@ -589,21 +590,37 @@ public function hookActionUpdateQuantity($params)
...
+
+             $result = Db::getInstance()->getRow($sql);
+
+            if ($this->customer_qty && $result['quantity'] >= $result['minimal_quantity']) {
+                MailAlert::sendCustomerAlert((int) $result['id_product'], (int) $params['id_product_attribute']);
+            }
+        }
+        else{
+            if ($this->customer_qty && $quantity >= $minimal_quantity) {
+                    MailAlert::sendCustomerAlert((int) $product->id, (int) $params['id_product_attribute']);
```
Comment (PierreRambaud) [inline ps_emailalerts.php]: You need to fix indentation

File: ps_emailalerts.php [hunk 2]
```diff
@@ -589,21 +590,37 @@ public function hookActionUpdateQuantity($params)
             }
         }
 
-        if ($this->customer_qty && $quantity > 0) {
-            MailAlert::sendCustomerAlert((int) $product->id, (int) $params['id_product_attribute']);
+        if($product_has_attributes) {
```
Comment (PierreRambaud) [inline ps_emailalerts.php]:
```suggestion
        if ($product_has_attributes) {
```

File: ps_emailalerts.php [hunk 3]
```diff
@@ -589,21 +590,37 @@ public function hookActionUpdateQuantity($params)
...
+            		ON sa.id_product_attribute = pa.id_product_attribute
+			WHERE sa.id_product_attribute = '. $id_product_attribute;
+
+             $result = Db::getInstance()->getRow($sql);
+
+            if ($this->customer_qty && $result['quantity'] >= $result['minimal_quantity']) {
+                MailAlert::sendCustomerAlert((int) $result['id_product'], (int) $params['id_product_attribute']);
+            }
+        }
+        else{
```
Comment (PierreRambaud) [inline ps_emailalerts.php]:
```suggestion
        } else {
```

**Comment by colinegin** [comment]
Hello @Rmagic96 , you got some feedbacks from the dev team, are you able to make the required changes ?

Thanks !

**Comment by Rmagic96** [comment]
> Hello @Rmagic96 , you got some feedbacks from the dev team, are you able to make the required changes ?
> 
> Thanks !

Sure, its my first commit here and didn’t know i had to make the changes.

I will make those changes asap (today/tomorrow) and will make a new commit.

Thanks for telling me!

**Comment by hibatallahAouadni** [comment]
@PrestaShop/prestashop-core-developers could someone please check this PR, it seems the author has made some changes but the PR still has the **`Waiting for author`** label :wink:

**Comment by Progi1984** [comment]
@Rmagic96 Could you rebase your PR ?

**Comment by kpodemski** [comment]
Hello @Rmagic96 

I'm closing this PR since we have no news from you and another user created PR which is already in QA validation:
https://github.com/PrestaShop/ps_emailalerts/pull/85

Thank you for your work and I hope to see you again in other contribution!

**Pull request closed without merging**